cmd/bcb.c

```diff
@@ -48,6 +49,9 @@ static int bcb_is_misused(int argc, char *const argv[])
 
 	switch (cmd) {
 	case BCB_CMD_LOAD:
+		if (argc != 3 && argc != 4)
+			goto err;
+		break;
 	case BCB_CMD_FIELD_SET:
 		if (argc != 3)
```

Comment:
argc != 4 - do we need this check here as well?

Reply:
yeah, we need to check both 3 & 4 old and new API